pkg/crc/cluster/cluster.go

```diff
@@ -67,6 +67,28 @@ func getcertExpiryDateFromVM(sshRunner *ssh.Runner) (time.Time, error) {
 	return certExpiryDate, nil
 }
 
+func CheckAggregatorClientCAValidity(sshRunner *ssh.Runner) (bool, error) {
+	certExpiryDate, err := getAggregatorClientCAExpiryDate(sshRunner)
+	if err != nil {
+		return false, err
+	}
+	return time.Now().After(certExpiryDate), nil
+}
+
+func getAggregatorClientCAExpiryDate(sshRunner *ssh.Runner) (time.Time, error) {
+	certExpiryDate := time.Time{}
+	certExpiryDateCmd := `date --date="$(sudo openssl x509 -in /etc/kubernetes/static-pod-resources/kube-apiserver-certs/configmaps/aggregator-client-ca/ca-bundle.crt -noout -enddate | cut -d= -f 2)" --iso-8601=seconds`
```

Comment:
Feel like we should have a helper which takes a `sshRunner` and a `path` as arguments and returns a cert expiry date (or a boolean indicating if the cert is expired).

Reply:
Yes I'm waiting for https://github.com/code-ready/crc/pull/1532 to get this in a better shape. I opened it collect feedback on "is it a good thing or not to do?"

Reply:
done